app/models/alchemy/element.rb

```diff
@@ -178,6 +178,8 @@ def all_from_clipboard_for_parent_element(clipboard, parent_element)
 
         all_from_clipboard(clipboard).where(name: parent_element.definition["nestable_elements"])
       end
+
+      deprecate :available, :none, "Please use Alchemy::Element#public and the Enumerable module instead.", deprecator: Alchemy::Deprecation
```

Comment:
The deprecation message is not very good. 

1. It's Alchemy::Element.published (class vs. instance method)
2. Why are you deprecation it with `:none`. Should it be `published`
3. We should be more precise what Enumerable module you are talking about. ElementRepository? Or just Rubys Enumerable?